src/coreclr/src/inc/corinfo.h

```diff
@@ -1344,6 +1345,7 @@ struct CORINFO_RUNTIME_LOOKUP
     bool                    testForFixup;
 
     SIZE_T                  offsets[CORINFO_MAXINDIRECTIONS];
+    WORD                    sizeOffset;
```

Comment:
You will need the same change in crossgen2 src\coreclr\src\tools\Common\JitInterface\CorInfoTypes.cs

Reply:
Fixed